src/js/themes/media-theme-template.js

```diff
@@ -0,0 +1,117 @@
+import { defineCustomElement } from '../utils/defineCustomElement.js';
+import MediaTheme from './media-theme.js';
+import { InnerTemplatePart, TemplateInstance, AttrPart } from './template-parts.js';
+
+class MediaThemeTemplate extends MediaTheme {
+  constructor() {
+    super();
+  }
+
+  connectedCallback() {
+    this.template = this.getRootNode().querySelector(`#${this.getAttribute('template')}`);
+
+    this.templateInstance = new TemplateInstance(
+      this.template,
+      this.props,
+      mediaThemeProcessor
+    );
+
+    this.shadowRoot.append(this.templateInstance);
+
+    const observer = new MutationObserver(() => this.render());
+    observer.observe(this, { attributes: true });
+  }
+
+  get props() {
+    const props = {};
+    for (let attr of this.attributes) {
+      if (attr.value != null) {
+        props[camelCase(attr.name)] = attr.value === '' ? true : attr.value;
+      } else {
```

Comment:
issue(blocking): removed attributes will not be members of the `attributes` object, so this case will never be met. One possible alternative is to instead make a the `props` object a `proxy` that evaluates on the key:value pairs on the fly.

Reply:
yes, but the change in attributes will be detected and the parts will be checked. 
if `props[partexpression]` is nullish it will be be deleted or handled